sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -155,7 +162,8 @@ public Duration convert(String value) {
         QueryTopOrderby,
         Mixed,
         ReadMyWrites,
-        ReadThroughputWithMultipleClients;
+        ReadThroughputWithMultipleClients,
+        CtlWorkload;
```

Comment:
shouldn't this be CTLWorkload?

Reply:
yes CTL is short name Itself , but CtlWorkload is more consistent with other workload camelcase